storage/kubernetes/storage.go

```diff
@@ -148,6 +148,12 @@ func (cli *client) registerCustomResources(useTPR bool) (ok bool) {
 			resourceName = r.ObjectMeta.Name
 		} else {
 			r := customResourceDefinitions[i]
+			var i interface{}
+			cli.logger.Infof("checking if resource %s has been created already...", r.ObjectMeta.Name)
```

Comment:
nit: if you're going to log this, I think it'd be good to log the error as well.

Right now these logs look like:

```
checking if resource %s has been created already...
create custom resource %s
```

Can you catch the error from the list and print that too?

```go
if err := cli.list(r.Spec.Names.Plural, &i); err == nil {
    // ...
} else {
    cli.logger.Infof("failed to list custom resource %s, attempting to create: %v", r.ObjectMeta,Name, err)
}
```